### PR TITLE
Micro metrics datadog

### PR DESCRIPTION
**Pull request by arunbcodes**
created datadog plug-in for micro-server and fixed a bug in the micro-event-metrics plugin

### CODE REVIEW

**Comment by johnmcclean-aol** [comment]
This is really cool Arun! 

Do you have time to write a readme.md for this plugin too? Something that would explain to users what it does and how to use it? (You can copy the structure from one of the other plugins).

**Comment by arunbcodes** [comment]
I have added a readme file as part of this commit.. can you take a look at this https://github.com/aol/micro-server/pull/245/files#diff-6ba9e18256ae90289a3d5253b568a69cR1 and let me know if that looks fine.

File: micro-metrics-datadog/readme.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        <version>x.yz</version>
+     </dependency>
+     
+Gradle
+
+    compile 'com.aol.microservices:micro-metrics-datadog:x.yz'
+    
+ # Configuring datadog metrics Reporters
+ 
+ By default we report to the datadog every second. with the tag stage:dev. To modify these settings, change the values in the application.properties file.
```
Comment (tonyfinn) [inline micro-metrics-datadog/readme.md]: Might be worth listing (at least the most common) options that can be changed in application.properties

Comment (arunbcodes) [inline micro-metrics-datadog/readme.md]: Have added some more information to that now.

**Pull request merged**